src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -12,6 +12,13 @@ export default class AdditionalInfo extends React.Component {
     this.state = { open: false };
   }
 
+  handleKeyDown = event => {
+    if (event.key === '' || event.keyCode === 32) {
```

Comment:
Should the `key` be a space instead of an empty string?

```suggestion
    if (event.key === ' ' || event.keyCode === 32) {
```

Also while I was reading docs, I came across [this one](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)/ It looks like `keyCode` is deprecated. Can we use `code` instead?

Reply:
Just ditched the keyCode entirely and went with your recommendation of key with a blank space (I didn't see the option to commit here, so pushed it up with my other changes).